src/h3/_internal_api.py

```diff
@@ -2,7 +2,7 @@
 
 """
 This module should serve as the interface between the C/Cython code and
-the Python code. That is, this module should import all the Cython functions we
+the Python code, i.e., an internal API. That is, this module should import all the Cython functions we
```

Comment:
nit suggestion: "... the Python code, that is, it is an internal API. This module should import all the Cython functions we ..."

Reply:
updated